ExpSwift/Classes/Exp.swift

```diff
@@ -1057,7 +1149,20 @@ private func refreshAuthToken(_ result:Auth){
             refreshAuthToken(result)
         }
     }
+}
 
+/**
+ Refresh Auth Token Recursive with Timeout
+ */
+private func refreshAuthToken(_ expirationToken:String){
```

Comment:
expirationToken should be renamed to be more clear that it's a timeout, not a token.